### PR TITLE
Make entry sample more minimal

### PR DESCRIPTION
**Pull request by Manvi-Agrawal**
Fixes https://github.com/microsoft/qsharp/pull/1579/files#r1632066063

### CODE REVIEW

File: samples/language/GettingStarted.qs [hunk 1]
```diff
@@ -5,8 +5,5 @@
 /// This is a minimal Q# program that can be used to start writing Q# code.
 namespace MyQuantumProgram {
 
-    @EntryPoint()
-    operation Main() : Unit {
-        // TODO: Write your Q# code here.
-    }
+    operation Main() : Unit {}
```
[the remark this answers is not in the document]
Comment (swernli) [reply samples/language/GettingStarted.qs:8]: I agree, keeping the comment seems worthwhile.

[the remark this answers is not in the document]
Comment (swernli) [reply samples/language/GettingStarted.qs]: This was fixed in #1652, I believe. So a merge from main should address the missing code lenses. If it does, you can close the new issue.

**Comment by swernli** [comment]
I think this is the right set of changes, but we should hold off on merging until we are closer to a release. Since the samples also act as a live set of references for users, we don't want to update samples with newer syntax or patterns that don't work in currently released version. This should get merged either right before or right after our next release, which will be the first to include the support for skipping `@EntryPoint` in favor of `Main`.

File: samples/language/GettingStarted.qs [hunk 2]
```diff
@@ -5,7 +5,6 @@
 /// This is a minimal Q# program that can be used to start writing Q# code.
 namespace MyQuantumProgram {
```
Comment (DmitryVasilevsky) [inline samples/language/GettingStarted.qs:6]: We can remove explicit namespace.

**Review by tcNickolas — APPROVED** [review]

**Review by DmitryVasilevsky — APPROVED** [review]

**Pull request merged**